pkg/controller/mongodb/testdata/change_data_volume.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: mongodb.com/v1
+kind: MongoDB
+metadata:
+  name: change-data-volume-mdb
+spec:
+  members: 3
+  type: ReplicaSet
+  version: "4.2.6"
+  persistent: true
+  security:
+    authentication:
+      modes: ["SCRAM"]
+  users:
+    - name: my-user
+      db: admin
+      passwordSecretRef:
+        name: my-user-password
+      roles:
+        - name: clusterAdmin
+          db: admin
+        - name: userAdminAnyDatabase
+          db: admin
+  statefulSet:
+    spec:
+      volumeClaimTemplates:
+        - metadata:
+            name: data-volume
+          spec:
+            accessModes: [ "ReadWriteOnce", "ReadWriteMany" ]
+            resources:
+              requests:
+                storage: 50Gi
```

Comment:
I see github here warning me about no newline at the end of file. Do we care?

Reply:
I'll fix it!